build_library/qemu_template.sh

```diff
@@ -26,6 +27,15 @@ Options:
     -c FILE     Config drive as an iso or fat filesystem image.
     -a FILE     SSH public keys for login access. [~/.ssh/id_{dsa,rsa}.pub]
     -p PORT     The port on localhost to map to the VM's sshd. [2222]
+    -I FILE     Set a custom image file.
+    -M MB       Set VM memory in MBs.
+    -T DIR      Add a software TPM2 device through swtpm which stores secrets
+                and the control socket to the given directory. This may need
+                some configuration first with 'swtpm_setup --tpmstate DIR ...'.
```

Comment:
Maybe add here:

```
                Add a software TPM2 device through swtpm which stores secrets
                and the control socket to the given directory. 
                See: https://github.com/stefanberger/swtpm/wiki/Certificates-created-by-swtpm_setup#tpm-2-certificates.'

```

Reply:
Thanks for the link, that wiki is the best place to document more complex setups and how distros vary